contrib/dependencies.txt

```diff
@@ -0,0 +1,34 @@
+# Fedora 31 dependencies for building podman
```

Comment:
I would not specify the Fedora 31, this will just be a hassle in the future.  Change this to just Fedora

Reply:
Fixed. )